### PR TITLE
feature/48815 user interface for activating feature flags within the settings

### PR DESCRIPTION
**Pull request by wielinde**
[#48815] User interface for activating feature flags within the settings

https://community.openproject.org/work_packages/48815

### CODE REVIEW

**Comment by klaustopher** [comment]
Translation for the confirmation dialog is missing:

Bildschirmfoto 2023-06-27 um 18 09 24

Rest looks fine and work as intended

File: config/locales/en.yml [hunk 1]
```diff
@@ -2726,6 +2727,10 @@ en:
       first_week_of_year_text: >
         Select the date of January that is contained in the first week of the year.
         This value together with first day of the week determines the total number of weeks in a year.
+    experimental:
+      save_confirmation: Caution! Risk of data loss! Only activate experimental features if you do not mind breaking your OpenProject installation and all its data.
```
Comment (mrmir) [inline config/locales/en.yml]: "... installation and **losing** all **of** its data.", maybe?

File: config/locales/en.yml [hunk 2]
```diff
@@ -2726,6 +2727,10 @@ en:
...
+      warning_toast: Feature flags are settings that activate features that are still under development. They shall only be used for testing purposes. They shall never be activated on OpenProject installations holding important data. These features will very likely corrupt your data. Use it on your own risk.
```
Comment (mrmir) [inline config/locales/en.yml]: "Use **them at** your own risk." sounds better to me, since everything else above it is plural.

**Review by klaustopher — APPROVED** [review]
👍

**Pull request merged**